packages/flutter/example/linux/flutter/ephemeral/.plugin_symlinks/connectivity_plus_linux

```diff
@@ -1 +1 @@
-/Users/rodrigo/flutter/.pub-cache/hosted/pub.dartlang.org/connectivity_plus_linux-1.0.2/
\ No newline at end of file
+/Users/rodrigo/flutter/.pub-cache/hosted/pub.dartlang.org/connectivity_plus_linux-1.3.0/
```

Comment:
why is there your personal, local path in the source code?

Reply:
This file should be in `.gitignore`

I have to validate

Reply:
New created flutter projects have (only) `flutter/ephemeral` in the .gitignore of the linux directories.